man/top_articles.Rd

```diff
@@ -4,22 +4,21 @@
 \alias{top_articles}
 \title{Retrieve Data on Top Articles}
 \usage{
-top_articles(project = "en.wikipedia", platform = "all", year = "2015",
-  month = "10", day = "01", reformat = TRUE, ...)
+top_articles(project = "en.wikipedia", platform = "all",
+  start = as.Date("2015-10-01"), granularity = "daily", reformat = TRUE,
```

Comment:
Part of the usefulness of an API wrapper should be that it allows R users to keep things as far as possible in a consistent, R vernacular and leverage existing data types.